CONTRIBUTING.md

```diff
@@ -351,6 +351,226 @@ The third schema is `third.schema.json`, it extends both `second`, and transitiv
+
+Details on using and defining schema descriptors may be found in [SCHEMA_DESCRIPTORS.md](./SCHEMA_DESCRIPTORS.md)
+
+Schema descriptors are extensible, and new descriptors may be creating by defining a new URI value and using it in
+the `@type` property of the descriptor object. Readers should ignore descriptors they do not understand.
+
+Schema descriptors are defined in XDM using the `SchemaDescriptor` schema.
+
+#### Defining Schema Relationships
+
+While schema descriptors can be used to define metadata about a single schema, they are also common used to describe relationships between schemas. This mechanism can be used to link schemas together at the property level, defining the equivalent of "foreign key" relationships in a relational database.
+
+The following relationship types are defined by XDM:
+
+* `xdm:descriptorOneToOne`: describes a 1:1 relationship between a source schema and a destination schema
+* `xdm:descriptorOneToMany`: describes a 1:m relationship between a source schema and a destination schema
+* `xdm:descriptorManyToMany`: describes an m:n relationship between a source schema and a destination schema
+
+These relationships are defined in XDM using the RelationshipDescriptor schema.
+
+#### Update Policies
+
+Data described by an XDM schema may change over time, and as such a data object may reflect an update of a previous instance of that object. There are different ways that an update may be handled, and this way depends both on the nature of the data and the specific application it is being used for.
+
+XDM defines a schema descriptor of type `xdm:descriptorUpdatePolicy`, which describes several common methods of handling an update:
+
+* `xdm:updateMerge`: the data in the new object should be merged into the existing object; the method by which a merge is applied is defined by the application
+* `xdm:updateReplace`: the new data object should replace the existing data object
+* `xdm:updateTimeSeries`: the data is time series data, and the new object should be logged/collected without changing any existing data
+
+Update policies are defined using the UpdatePolicyDescriptor schema.
+
+#### Other Supported Schema Descriptors
+
+A number of additional schema descriptors are defined by XDM:
+
+* `xdm:descriptorIdentity`: allows a property in a schema to be used as an identity, even if it does not conform to the Identity schema.
+* `xdm:descriptorPrimaryKey`: allows a property other than `@id` to be flagged as the primary key for a schema
+* `xdm:descriptorInstantiable`: allows a schema to be flagged as 'instantiable', which may be used to differentiate schemas that define primary business objects versus supporting schemas intended to be embedded in another schema.
```

Comment:
This seems to be simply the negation of `meta:abstract`, so I'm not sure we need it.